providers/aws/fargate.toml

```diff
@@ -32,6 +32,7 @@ ExecutionRoleArn = ""
 
 # Amazon CloudWatch log group name used to store container logs. Optional.
 # If omitted, container logs will not be available.
+# If specified, execution role is required.
```

Comment:
Minor: We can provide more info here:
"If specified, an execution role with access to CloudWatch logs is required."

Reply:
OK, I'll update.